website/docs/docs/build/snapshots.md

```diff
@@ -587,6 +587,16 @@ Snapshot results:
 
 </details>
 
+<VersionBlock firstVersion="1.9">
+
+### Define column names in snapshot table
```

Comment:
I don't think this is the correct language.. something more like "customize the names of the dbt-generated meta fields in your snapshot tables"